.test/unit/test.cpp

```diff
@@ -1257,6 +1259,39 @@ TEST_CASE( "Hypercomplex: MPFR lib test", "[unit]" ) {
             clear_mpfr_memory();
         }
     }
+
+    SECTION( "Revision test" ) {
+        mpfr_t A[8];
```

Comment:
maybe you want to set mpfr precision here to be on the safe side.

Reply:
This one is for sure OK; Catch2 tests work in a tree-structure; i.e. test cases are evaluated as separate branches with the most-nested cases being leaves. all code from the internal nodes are executed before reaching the leaf. The section above is a subsection of the one in line `521`, right there I set the precision and this gets passed to all branches downstream.